README.md

```diff
@@ -47,6 +47,7 @@ On the backend, the best thing you can do is just build a module or two. We coul
 - [Suraj Kumar paul](https://certification.acquia.com/user/675520) - Passed exam 2019-08-07
 - [Kristiyan Nikolov](https://certification.acquia.com/user/1657941) - Passed exam 2019-08-30
 - [Hemant Joshi](https://certification.acquia.com/user/1659171) - Passed exam 2019-09-29
+- [Ravi Kumawat](https://certification.acquia.com/user/1658696) - Passed exam 2019-08-21
```

Comment:
@ravikhetri This list is sorted by date. Would you mind redoing this in the correct order?

Reply:
Thanks.
I have added new pull request as per date.
https://github.com/WidgetsBurritos/d8-studyguide/pull/124